dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/Constants.java

```diff
@@ -95,10 +95,10 @@
     String INVOCATION_KEY = "invocation";
     String SERIALIZATION_ID_KEY = "serialization_id";
 
-    String H2_SETTINGS_HEADER_TABLE_SIZE_KEY = "h2.settings.header-table-size";
-    String H2_SETTINGS_ENABLE_PUSH_KEY = "h2.settings.enable-push";
-    String H2_SETTINGS_MAX_CONCURRENT_STREAMS_KEY = "h2.settings.max-concurrent-streams";
-    String H2_SETTINGS_INITIAL_WINDOW_SIZE_KEY = "h2.settings.initial-window-size";
-    String H2_SETTINGS_MAX_FRAME_SIZE_KEY = "h2.settings.max-frame-size";
-    String H2_SETTINGS_MAX_HEADER_LIST_SIZE_KEY = "h2.settings.max-header-list-size";
+    String H2_SETTINGS_HEADER_TABLE_SIZE_KEY = "dubbo.rpc.tri.header-table-size";
+    String H2_SETTINGS_ENABLE_PUSH_KEY = "dubbo.rpc.tri.enable-push";
+    String H2_SETTINGS_MAX_CONCURRENT_STREAMS_KEY = "dubbo.rpc.tri.max-concurrent-streams";
+    String H2_SETTINGS_INITIAL_WINDOW_SIZE_KEY = "dubbo.rpc.tri.init-window-size";
```

Comment:
It's better to just change the prefix . `dubbo.rpc.tri.initial-window-size`

Reply:
ACK